Makefile

```
@@ -155,36 +156,51 @@ dist-ui: build-ui
 
 release: dist-ui build-server ## Build wheel file for release
 
-container-image: ## Build Elyra stand-alone container image
+elyra-image:
+	## Build Elyra stand-alone container image
 	@mkdir -p build/docker
 	cp etc/docker/elyra/Dockerfile build/docker/Dockerfile
 	cp etc/docker/elyra/start-elyra.sh build/docker/start-elyra.sh
-	DOCKER_BUILDKIT=1 docker build -t docker.io/$(IMAGE) -t quay.io/$(IMAGE) build/docker/ --progress plain
+	DOCKER_BUILDKIT=1 docker build -t docker.io/$(ELYRA_IMAGE) -t quay.io/$(ELYRA_IMAGE) build/docker/ --progress plain
 
-airflow-image: ## Build airflow image for use with Elyra
+publish-elyra-image: elyra-image
+	## Publish Elyra stand-alone container image
+    # this is a privileged operation; a `docker login` might be required
+	docker push docker.io/$(ELYRA_IMAGE)
+	docker push quay.io/$(ELYRA_IMAGE)
+
+airflow-image:
+	## Build airflow image for use with Elyra
 	DOCKER_BUILDKIT=1 docker build -t docker.io/$(ELYRA_AIRFLOW_IMAGE) -t quay.io/$(ELYRA_AIRFLOW_IMAGE) \
 	--build-arg AIRFLOW_NOTEBOOK_VERSION=$(AIRFLOW_NOTEBOOK_VERSION) etc/docker/airflow/ --progress plain
 
-kf-notebook-image: ## Build elyra image for use with Kubeflow Notebook Server
+publish-airflow-image: airflow-image
+	## Publish airflow image for use with Elyra
+	# this is a privileged operation; a `docker login` might be required
+	docker push docker.io/$(ELYRA_AIRFLOW_IMAGE)
+	docker push quay.io/$(ELYRA_AIRFLOW_IMAGE)
+
+kf-notebook-image:
+	## Build elyra image for use with Kubeflow Notebook Server
 	DOCKER_BUILDKIT=1 docker build -t docker.io/$(KF_NOTEBOOK_IMAGE) -t quay.io/$(KF_NOTEBOOK_IMAGE) \
 	etc/docker/kubeflow/ --progress plain
 
-container-images: container-image kf-notebook-image airflow-image ## build all container images
-	docker images $(IMAGE)
-	docker images quay.io/$(IMAGE)
+publish-kf-notebook-image: kf-notebook-image
+	## Publish elyra image for use with Kubeflow Notebook Server
+	# this is a privileged operation; a `docker login` might be required
+	docker push docker.io/$(KF_NOTEBOOK_IMAGE)
+	docker push quay.io/$(KF_NOTEBOOK_IMAGE)
+
+container-images: elyra-image kf-notebook-image airflow-image ## build all container images
```

Upcase the comment since all others are that way.
```suggestion
container-images: elyra-image kf-notebook-image airflow-image ## Build all container images
```